Spanner/tests/System/AdminTest.php

```diff
@@ -105,8 +106,10 @@ public function testDatabase()
         });
 
         $this->assertInstanceOf(Database::class, current($database));
-
         $this->assertTrue($db->exists());
+        // since we didn't pass any dialect while creation,
+        // it should equal GSSQL
+        $this->assertEquals($db->info()['databaseDialect'], DatabaseDialect::GOOGLE_STANDARD_SQL);
```

Comment:
This assertion is failing in tests.... it's returning `DatabaseDialect::DATABASE_DIALECT_UNSPECIFIED` instead.

I imagine this is WAD by the API.  If it's WAD, we could potentially set the `DatabaseDialect` to be `GOOGLE_STANDARD_SQL` by default in the `Database::create` function.

Reply:
This was failing in the emulator I think.
The latest Spanner product release contains this, so the system tests are passing.
I had to skip the emulator tests for this one as the emulator currently doesn't support PGSQL.

Reply:
I would really like to not skip this whole test just for the one assertion. This suggestion makes it clear this is a temporary fix, while still keeping the existing tests. Just skipping the tests entirely means that the problem will never be corrected. I imagine the emulator _should_ be fixed in the future, and this is essentially just for backwards compatibility. Speaking of, have you logged a bug so that the emulator team can fix this?

```suggestion
        $expectedDatabaseDialect = DatabaseDialect::GOOGLE_STANDARD_SQL;
        if ((bool) getenv("SPANNER_EMULATOR_HOST")) {
            // TODO: Remove this, as there is currently a bug in the emulator where the default database dialect is
            // not set to GOOGLE_STANDARD_SQL as expected. This is because the emulator currently does not 
            // support PGSQL
            $expectedDatabaseDialect = DatabaseDialect::DATABASE_DIALECT_UNSPECIFIED
        }
        $this->assertEquals($db->info()['databaseDialect'], $expectedDatabaseDialect);
```